$NetBSD: patch-languages_english_english.2,v 1.1 2021/01/24 15:53:02 nia Exp $

Remove spelling mistake.
https://gnats.netbsd.org/5181

Keep the hunk at its current release offset so GNV patch preserves filename
case on OpenVMS.

--- languages/english/english.2.orig
+++ languages/english/english.2
@@ -2078,7 +2078,6 @@
 auxotroph
 auxotrophic
 auxotrophy
-avaliable
 avascular
 avast
 avatar
